pkg/kube/kube.go

```diff
@@ -230,7 +230,7 @@ func (k *KubeClient) runKubectlCmdWithInput(args []string, input []byte) ([]byte
 	return out, nil
 }
 func (k *KubeClient) LabelExistsInCluster(ctx context.Context, label string) error {
-	cmd := []string{"get", "app", "-l", fmt.Sprintf("weave-gitops.weave.works/app-identifier=%s", label), "--all-namespaces"}
+	cmd := []string{"get", "app", "-l", fmt.Sprintf("wego.weave.works/app-identifier=%s", label), "--all-namespaces"}
```

Comment:
Should this be `app.WeGOAppIdentifierLabelKey`>

Reply:
It doesn't matter much because of the commit discussion this morning we will be removing this label